packages/kbn-dom-drag-drop/src/sass/drag_drop.scss

```diff
@@ -182,12 +167,20 @@ $reorderItemMargin: $euiSizeS;
   width: 100%;
   height: 100%;
   background: $euiColorLightestShade;
-  border-radius: 0;
-  &:first-child, &:first-child .domDragDrop__extraDrop {
+
+  .domDragDrop__extraDrop,
+  .domDragDrop__extraDrop:before {
+    border-radius: 0;
+  }
+
+  &:first-child .domDragDrop__extraDrop,
+  &:first-child .domDragDrop__extraDrop:before {
     border-top-left-radius: $euiSizeXS;
     border-top-right-radius: $euiSizeXS;
   }
-  &:last-child, &:last-child .domDragDrop__extraDrop {
+
+  &:last-child .domDragDrop__extraDrop,
+  &:last-child .domDragDrop__extraDrop:before {
     border-bottom-left-radius: $euiSizeXS;
     border-bottom-right-radius: $euiSizeXS;
   }
```

Comment:
Should these references to `$euiSizeXS` be changed to `$euiBorderRadiusSmall`?

Reply:
Aaah, I had the automerge set on 🤦🏼 I will add it to the next PR 👍🏼